### PR TITLE
Allow a user to download the GovWifi poster

### PR DESCRIPTION
**Pull request by georgeschena**

### CODE REVIEW

**Comment by Renny4Real** [comment]
Wondering if this should have a test around it?

**Comment by kjdchapman** [comment]
From a quick google, it looks like it's quite hard to test whether the download has happened - but maybe we should just test that the content ("Download a poster") is present on this page?

**Comment by Renny4Real** [comment]
There's a spec around uploading and downloading the MOU. 
Might get some inspiration from there as well?

File: app/views/home/index.html.erb [hunk 1]
```diff
@@ -44,7 +44,9 @@
     <div>
       <h3 class="govuk-heading-m"> 5. Check the connection works </h3>
       <p class="govuk-body"> Connect to your new GovWifi network, and sign in with an individual account. </p>
-      <details class="govuk-details">
+      <%= link_to "Download a poster to advertise GovWifi is available in your building(s)", asset_path('govwifi-poster.png'), { download: "" ,class: "govuk-link govuk-body-m"}  %>
```
Comment (kjdchapman) [inline app/views/home/index.html.erb]: I think this doesn't need the govuk-body-m class?
```suggestion
       <%= link_to "Download a poster to advertise GovWifi is available in your building(s)", asset_path('govwifi-poster.png'), { download: "" ,class: "govuk-link"}  %>
```

**Pull request merged**